### DISCUSSION TITLE
Nested m2o item creation sends GET instead of POST

### DISCUSSION THREAD

**Discussion by ryansann** [directus/directus#15912] [Archive]
Hi, I have a collection with a o2m alias field that displays related items. Everything seems to work fine with this in terms of creating and updating related items. The aliased collection also has a m2o field for selecting from a list of pre-defined values. The dropdown has a `+` next to it  indicating a new value can be added. When proceeding, it correctly states, `Creating an Item in <collection>`. However according to the API documentation, the wrong method is being used. A GET request is sent when it should be a POST for new item creation.

I'm attaching a screen capture that shows the flow I am talking about. It appears as though the Link Name item is created, but in fact a `GET` was sent for the new item, which since it doesn't exist, returns  a 403 forbidden.

Directus version: `9.14.5`

Posting here as a discussion because I'm trying to figure our if this is a user / configuration error or an actual bug. Any assistance would be appreciated, thank you!

https://user-images.githubusercontent.com/8824599/194438096-e8fd306a-8ede-4ce6-8057-ba91cb975986.mov

**Comment by br41nslug** [comment]
The created item is saved when the original item is saved but the video only shows the side drawer popup 😬 The GET you're seeing is probably the drawer trying to fetch its display information.

**Reply by ryansann** [reply]
Ahh you are right, thank you for the reply. I was expecting the closing of the drawer to trigger the nested item creation, but it does in fact get posted when saving the original item. Thanks @br41nslug
